src/msw/joystick.cpp

```diff
@@ -62,6 +181,9 @@ wxJoystick::wxJoystick(int joystick)
             {
                 /* Found the one we want, store actual OS id and return */
                 m_joystick = i;
+                m_thread = new wxJoystickThread(m_joystick);
+                m_thread->Create();
```

Comment:
This is unnecessary, `Run()` will call `Create()` on its own.

Reply:
Fixed.